### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by jacobwindsor**
This implements CI/CD using Gitlab with kubernetes integration. Template for the CI process is [here](https://github.com/ebi-ait/gitlab-ci-templates/blob/master/build-release-deploy.yml)

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,29 +1,30 @@
...
 
-
-build:
-  stage: build
-  script:
-    - ./gradlew
-    
-test:
-  stage: test
+Unit Test:
```
Comment (amnonkhen) [inline .gitlab-ci.yml:21]: Why is this one Title Case while all the others kebab-case?

Comment (jacobwindsor) [reply .gitlab-ci.yml:9]: Snake case if for the stage name and title case is for the job name. The job name is what shows in the pipeline UI so title case looks nicer. You can see https://github.com/ebi-ait/gitlab-ci-templates/blob/master/build-release-deploy.yml for full details of the jobs

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,29 +1,18 @@
...
-
-workflow:
-  rules:
-    - if: $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == 'dev'
-
-stages:
-  - build
-  - test
+include:
+  remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
```
Comment (aaclan-ebi) [inline .gitlab-ci.yml:2]: The GitLab setup should also be documented somewhere. Might be good to put in the same repo as this template!

**Review by aaclan-ebi — APPROVED** [review]
Really awesome job setting this up! The flow makes sense to me!

File: .gitlab-ci.yml (hunk 2 above)
Comment (jacobwindsor) [reply .gitlab-ci.yml:2]: Yes! Will do

Comment (jacobwindsor) [reply .gitlab-ci.yml:2]: Done

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**